<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTestClass</class>
 <widget class="QMainWindow" name="QtTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>客户端</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="2">
        <widget class="QTextBrowser" name="textBrowser_clientRecive"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_clientStatus">
         <property name="text">
          <string>未连接</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_linkStart">
         <property name="text">
          <string>连接</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>地址</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_linkClose">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>断开</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>接收</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton_sendToServer">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QTextEdit" name="textEdit_clientToServer"/>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEdit_linkAddress">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>测试</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>true</bool>
    </property>
    <addaction name="action2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action2">
   <property name="text">
    <string>毛刺检测</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
